scripts/_utils/utils.py

```diff
@@ -84,6 +91,50 @@ def verify_mimetype(file_url, mimetype_string, local=False):
         return False
 
 
+def is_ffmpeg_compatible(file_url) -> bool:
+    command = "ffmpeg -y -v error -i " + file_url + " /tmp/verify-ffmpeg.mp4"
+    err = subprocess.run(command.split(" "), capture_output=True).stderr
+    return err == b''
+
+
+def verify_media_integrity(file_url, mime, local) -> Tuple[bool, Union[str, None], bool]:
+    """
+    Verifies and fixes media
```

Comment:
How, why?  Details pelase. I'd prefer a novel than just a repeat of the function name

Reply:
For example, it is only looking for images. what kind of images? all of them? how about animated gifs?  I should not have to read the code to know what this method does, and what it returns.

Reply:
fixed in [ab06ea3](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/48/commits/ab06ea3ff440c835606d11ff5f6e2405f1d65554)